transmission_interface/include/transmission_interface/transmission_interface_exception.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 PAL Robotics S.L.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TRANSMISSION_INTERFACE__TRANSMISSION_INTERFACE_EXCEPTION_HPP_
+#define TRANSMISSION_INTERFACE__TRANSMISSION_INTERFACE_EXCEPTION_HPP_
+
+#include <exception>
+#include <string>
+
+namespace transmission_interface
+{
+class TransmissionInterfaceException : public std::exception
+{
+public:
+  explicit TransmissionInterfaceException(const std::string & message) : msg(message) {}
```

Comment:
Is there really a value in this exception type? I am not so sure…

Reply:
There is a bit. These exceptions are captured when loading and handled nicely with a print and returning an empty transmission.
Other type of expressions (aka "unmanaged" ones) will however crash the entire system.

While it's not bad I also don't think it's really necessary to have this. I'm happy to remove this (as I basically put it out of use already) in #642

Reply:
removed in #642